src/main/java/uk/gov/hmcts/reform/wacaseeventhandler/services/initiatetask/InitiationTaskHandler.java

```diff
@@ -0,0 +1,47 @@
+package uk.gov.hmcts.reform.wacaseeventhandler.services.initiatetask;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.core.annotation.Order;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.wacaseeventhandler.clients.WaWorkflowApiClientToInitiateTask;
+import uk.gov.hmcts.reform.wacaseeventhandler.domain.DmnStringValue;
+import uk.gov.hmcts.reform.wacaseeventhandler.domain.EvaluateDmnRequest;
+import uk.gov.hmcts.reform.wacaseeventhandler.domain.EvaluateDmnResponse;
+import uk.gov.hmcts.reform.wacaseeventhandler.domain.initiatetask.InitiateTaskDmnRequest;
+import uk.gov.hmcts.reform.wacaseeventhandler.domain.initiatetask.InitiateTaskDmnResponse;
+import uk.gov.hmcts.reform.wacaseeventhandler.services.CaseEventHandler;
+
+@Service
+@Order(3)
+@Slf4j
+public class InitiationTaskHandler implements CaseEventHandler {
+
+    private final WaWorkflowApiClientToInitiateTask apiClientToInitiateTask;
+
+    public InitiationTaskHandler(WaWorkflowApiClientToInitiateTask apiClientToInitiateTask) {
+        this.apiClientToInitiateTask = apiClientToInitiateTask;
+    }
+
+    @Override
+    public boolean canHandle() {
+        EvaluateDmnResponse<InitiateTaskDmnResponse> response = apiClientToInitiateTask.evaluateDmn(
+            "getTask_IA_Asylum",
```

Comment:
can we define this as static  as with the dmn updates this will be renamed to something else better update the variable in one place :)

Reply:
Yes, I plan to do this as part of the latest PRs. Thanks.